lib/reversetunnel/transport.go

```diff
@@ -393,6 +415,7 @@ func (p *transport) tunnelDial(r *sshutils.DialReq) (net.Conn, error) {
 	// exists, then exit right away this code may be running outside of a
 	// remote site.
 	if p.reverseTunnelServer == nil {
+		p.log.Warn("can't perform tunnel dial, reverseTunnelServer is nil")
```

Comment:
Need this? If needed, seems more suitable for debug level.
```suggestion
		p.log.Warn("Can't perform tunnel dial, reverseTunnelServer is nil.")
```

Reply:
Nope, this was me trying to figure out what was going wrong. Good catch.